TOC.md

```diff
@@ -484,14 +484,17 @@
     - [概述](/ticdc/ticdc-overview.md)
     - [安装部署](/ticdc/deploy-ticdc.md)
     - [运维管理](/ticdc/manage-ticdc.md)
-    - [故障诊断](/ticdc/troubleshoot-ticdc.md)
-    - [监控指标](/ticdc/monitor-ticdc.md)
-    - [报警规则](/ticdc/ticdc-alert-rules.md)
-    - [TiCDC Open API](/ticdc/ticdc-open-api.md)
-    - [TiCDC Open Protocol](/ticdc/ticdc-open-protocol.md)
-    - [TiCDC Canal-JSON Protocol](/ticdc/ticdc-canal-json.md)
-    - [TiCDC Avro Protocol](/ticdc/ticdc-avro-protocol.md)
-    - [将 TiDB 集成到 Confluent Platform](/ticdc/integrate-confluent-using-ticdc.md)
+    - 监控告警
+      - [监控指标](/ticdc/monitor-ticdc.md)
+      - [报警规则](/ticdc/ticdc-alert-rules.md)
+    - [故障处理](/ticdc/troubleshoot-ticdc.md)
+    - 参考指南
+      - [TiCDC Open API](/ticdc/ticdc-open-api.md)
+      - [TiCDC Open Protocol](/ticdc/ticdc-open-protocol.md)
+      - [TiCDC Canal-JSON Protocol](/ticdc/ticdc-canal-json.md)
+      - [TiCDC Avro Protocol](/ticdc/ticdc-avro-protocol.md)
```

Comment:
Please keep the order of Chinese consistent with that of the English version if no special considerations.
```suggestion
      - [TiCDC Avro Protocol](/ticdc/ticdc-avro-protocol.md)
      - [TiCDC Canal-JSON Protocol](/ticdc/ticdc-canal-json.md)
```